lib/iris/analysis/__init__.py

```diff
@@ -1309,7 +1313,12 @@ def _calc_percentile(data, percent, fast_percentile_method=False, **kwargs):
                 "ignore",
                 "Warning: 'partition' will ignore the 'mask' of the MaskedArray.",
             )
-            result = np.percentile(data, percent, axis=-1)
+            pc_kwargs = dict(axis=-1)
+            if method is not None:
+                pc_kwargs["method"] = method
```

Comment:
This feels kinda ugly.  Is there a better way?

Reply:
I didn't think so, then I started wondering what happens if `scipy`'s `mquantiles` starts accepting a method argument, and now I think it might be better to pop `alphap` and `betap` out of the kwargs then pass the remainder wholesale to `np.percentile`?

Reply:
Ooh, that seems nice